tests/installed/destructive/staged-deploy.yml

```diff
@@ -113,5 +113,22 @@
   environment:
     commit: "{{ rpmostree_status['deployments'][0]['checksum'] }}"
 
+- name: Previous version is retained during staged
+  shell: |
+    set -xeuo pipefail
+    ostree --repo=/ostree/repo rfs --create nonstaged-deploy "${commit}"
+    ostree admin deploy nonstaged-deploy
+    ostree admin status > status.txt
+    grep -vqFe '(staged)' status.txt
+    ostree admin deploy --stage --retain-previous staged-deploy
```

Comment:
`--retain-previous-version`

Though... note there is no CI currently testing this. But we might make use of these tests (even if we lift them out of the playbooks) once we rework CI here. For this PR, we'll have to make sure we test it manually for now (and at least we'll get some CI coverage in the rpm-ostree testsuite once it makes use of it).

Here are some steps on how to do that using cosa: https://github.com/coreos/coreos-assembler/blob/master/README-devel.md#using-overrides

Reply:
Fixed that and changed the test to look more like what I was doing when testing it manually.